src/gui/masterkey/PasswordEditWidget.ui

```diff
@@ -52,13 +58,20 @@
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="PasswordEdit" name="repeatPasswordEdit">
+       <property name="accessibleName">
+        <string>Repeat password field</string>
```

Comment:
Does it make sense to add "field" to accessibility descriptions?

Reply:
Yes it is convention per accessibility lab